device-types/Cisco/C9200CX-12T-2X2G.yaml

```diff
@@ -0,0 +1,53 @@
+---
+manufacturer: Cisco
+model: Catalyst C9200CX-12T-2X2G
+part_number: C9200CX-12T-2X2G
+slug: cisco-c9200cx-12t-2x2g
```

Comment:
"Could not resolve id of device_type: c9200cx-12p-2x2g"
    netbox.netbox.netbox_device:
      data:
        name: "{{ ansible_facts['net_hostname'] }}"
        device_type: "{{ ansible_facts['net_model'] }}"


From sh ver:
Model Number                       : C9200CX-12P-2X2G

slug for c9200cx*
cisco-c9200cx-12p-2x2g
cisco-c9200cx-12p-2xgh
cisco-c9200cx-12t-2x2g

can slug be adjusted to match device model without cisco-